File: easy/Solve Me First/solveMeFirst.java
import java.util.*;

public class Solution {

    static int solveMeFirst(int a, int b) {
        int sum = 0;
        if (a >= 1 && a <= 1000) {
            if (b >= 1 && b <= 1000) {
                sum = a + b;
            }
        }
        return sum;
      // Hint: Type return a+b; below 
	}

  
   public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int a;
        a = in.nextInt();
        int b;
        b = in.nextInt();
        in.close();
        int sum;
        sum = solveMeFirst(a, b);
        System.out.println(sum);
	}
}

File: easy/Solve Me First/solveMeFirst.kt
import java.util.*

fun solveMeFirst(a: Int, b: Int): Int {
    var sum = 0;
    if (a > 0 && a <= 1000 ) {
        if (b > 0 && b <= 1000) {
        sum = a + b;
        }
    }
    return sum;
}

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)
    val num1 = sc.nextInt()
    val num2 = sc.nextInt()
    val sum = solveMeFirst(num1, num2)
    println(sum)
}
